tox.ini

```diff
@@ -0,0 +1,10 @@
+[tox]
+envlist = py35,py36,py37
+
+[testenv]
+deps =
+    pytest
+    pytest-cov
+commands =
+    pip install .
+    pytest --cov=chunked_scatter
```

Comment:
You could consider adding a `[testenv:py3-lint]` section. Here you can put all the linters you want to run.